source/shared/cpp/ObjectModel/HostConfig.h

```diff
@@ -132,9 +132,9 @@ namespace AdaptiveSharedNamespace
 
     struct ImageSizesConfig
     {
-        unsigned int smallSize = 80;
-        unsigned int mediumSize = 120;
-        unsigned int largeSize = 180;
+        unsigned int smallSize = 40;
```

Comment:
>unsigned int smallSize = 40; [](start = 8, length = 28)

was it a bug that these were the old values? should this be considered a breaking change for hosts that don't specify image sizes in their config?

Reply:
I shouldn't have made this change. I did it locally because the sizes that were in here didn't "look good" to me, but the existing values are the documented defaults and i should have reverted before sending the PR. I'll undo this and update the affected test files.

---
In reply to: [290044941](https://github.com/microsoft/AdaptiveCards/pull/3018#discussion_r290044941) [](ancestors = 290044941)

Reply:
hooray for code review! :)

---
In reply to: [290054705](https://github.com/microsoft/AdaptiveCards/pull/3018#discussion_r290054705) [](ancestors = 290054705,290044941)